newIDE/app/src/MainFrame/EditorContainers/HomePage/BuildSection/ExamplesLine.js

```diff
@@ -0,0 +1,55 @@
+// @flow
+import * as React from 'react';
+import { type ExampleShortHeader } from '../../../../Utils/GDevelopServices/Example';
+import { Column } from '../../../../UI/Grid';
+import Text from '../../../../UI/Text';
+import { LineStackLayout } from '../../../../UI/Layout';
+import FlatButton from '../../../../UI/FlatButton';
+import { Trans } from '@lingui/macro';
+import Add from '../../../../UI/CustomSvgIcons/Add';
+import ExamplesGrid from './ExamplesGrid';
+import { prepareExamples } from '../../../../AssetStore/ExampleStore';
+
+type ExamplesLineProps = {|
+  exampleShortHeaders: Array<ExampleShortHeader>,
+  onExpand: () => void,
+  onOpen: (exampleShortHeader: ExampleShortHeader) => void,
+|};
+
+const ExamplesLine = ({
```

Comment:
I'm not sure if I'm right or if I'm bad at english but `ExamplesRow` sound better to me.

Reply:
you're totally right